src/e2e-tests/__tests__/saving-preferred-attributions-globally.test.ts

```diff
@@ -0,0 +1,268 @@
+      packageInfo1,
+    );
+    await attributionDetails.assert.commentIs(preferLocallyComment);
+  });
+});
+
+test('marks and unmarks an attribution as preferred globally if user navigates away and saves', async ({
+  attributionDetails,
+  changePreferredStatusGloballyPopup,
+  notSavedPopup,
+  projectStatisticsPopup,
+  resourceBrowser,
+  resourceDetails,
+  menuBar,
+  topBar,
+}) => {
+  await projectStatisticsPopup.close();
+  await menuBar.toggleQaMode();
+  const preferLocallyCopyright = faker.lorem.sentence();
+
+  await test.step('prefer attribution locally', async () => {
+    await resourceBrowser.goto(resourceName1);
+    await resourceDetails.attributionCard.assert.preferredIconIsHidden(
+      packageInfo1,
+    );
+    await attributionDetails.copyright.fill(preferLocallyCopyright);
+    await attributionDetails.openHamburgerMenu();
+    await attributionDetails.hamburgerMenu.markAsPreferred.click();
+    await topBar.gotoAttributionView();
+    await notSavedPopup.assert.isVisible();
+
+    await notSavedPopup.saveButton.click();
+    await changePreferredStatusGloballyPopup.assert.isHidden();
+
+    await topBar.gotoAuditView();
+    await resourceBrowser.goto(resourceName1);
+    await resourceDetails.attributionCard.assert.preferredIconIsVisible(
+      packageInfo1,
+    );
+    await attributionDetails.assert.copyrightIs(preferLocallyCopyright);
+  });
+
+  await test.step('prefer attribution globally', async () => {
+    await resourceBrowser.goto(resourceName2);
+    await resourceDetails.attributionCard.assert.preferredIconIsHidden(
+      packageInfo1,
+    );
+    await attributionDetails.openHamburgerMenu();
+    await attributionDetails.hamburgerMenu.markAsPreferred.click();
+    await resourceBrowser.goto(resourceName3);
+    await notSavedPopup.assert.isVisible();
+
+    await notSavedPopup.saveGloballyButton.click();
+    await changePreferredStatusGloballyPopup.assert.showsMarkAsPreferredWarning();
+
+    await changePreferredStatusGloballyPopup.okButton.click();
+    await resourceDetails.attributionCard.assert.preferredIconIsVisible(
+      packageInfo1,
+    );
+  });
+
+  await test.step('unmark preferred globally', async () => {
+    await resourceDetails.attributionCard.assert.preferredIconIsVisible(
+      packageInfo1,
+    );
+
+    await attributionDetails.openHamburgerMenu();
+    await attributionDetails.hamburgerMenu.unmarkAsPreferred.click();
+    await resourceBrowser.goto(resourceName2);
+    await notSavedPopup.assert.isVisible();
+
+    await notSavedPopup.saveGloballyButton.click();
+    await changePreferredStatusGloballyPopup.assert.showsUnmarkAsPreferredWarning();
+
+    await changePreferredStatusGloballyPopup.okButton.click();
+    await resourceDetails.attributionCard.assert.preferredIconIsHidden(
+      packageInfo1,
+    );
+  });
+});
+
+test('popup is only shown if necessary and undo reverts preferrence status only', async ({
```

Comment:
```suggestion
test('popup is only shown if necessary and undo reverts preference status only', async ({
```

...but this test will change if we no longer actually undo the operation.